resources/lang/en/validation.php

```diff
@@ -141,7 +141,8 @@
         ],
         'room' => [
             'already_member' => 'The user is already member of the room.'
-        ]
+        ],
+        'password' => 'The password must contain at least one character from the following four categories: Uppercase letter (A - Z), lowercase letter (a - z), number (0 - 9), non-alphanumeric character (for example: !, $, #, or %).'
```

Comment:
The password must contain at least one character from each of the following four categories: